modules/azuredevops/project/buildingblock/README.md

```diff
@@ -0,0 +1,161 @@
+---
+name: Azure DevOps Project
+supportedPlatforms:
+  - azuredevops
```

Comment:
initialy the hub website should be extendend with the new platform otherwise it will not shown in hub

Reply:
done